todoman/model.py

```diff
@@ -91,9 +93,12 @@ def __init__(self, todo=None, filename=None, mtime=None, new=False):
 
         if new:
             now = datetime.now(self._localtimezone)
+            config = load_config()
```

Comment:
This is not a good place to do this, particularly because the config is already read from disk, and calling `load_config` again parses it again.

I suspect that the `Todo` class just shouldn't set a `due` date, and that the entire logic belongs into the `cli` module (particularly into the `new` command).